fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PruneScanColumnRule.java

```diff
@@ -61,13 +61,19 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
                 scanOperator.getColRefToColumnMetaMap().keySet().stream().filter(requiredOutputColumns::contains)
                         .collect(Collectors.toSet());
         outputColumns.addAll(Utils.extractColumnRef(scanOperator.getPredicate()));
-
+        boolean canUseAnyColumn = false;
```

Comment:
Why add this for olap table？

Reply:
We can set it, but we don't need to use it.